tests/test_django_default_settings.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from django.conf import global_settings as default
+
+import cbs
+
+
+class MySettings(object):
+    PROJECT_NAME = 'tests'
+
+    @property
+    def INSTALLED_APPS(self):
+        return default.INSTALLED_APPS + ['test',]
```

Comment:
If you look at the history of this test I think you'll find it used to try a different setting... to ensure it actually extended the original.